### ISSUE TITLE
Add root saturation function

### ISSUE THREAD

**Issue by wd60622** [pymc-labs/pymc-marketing#702]
I liked the idea of the root saturation that came up in #614 

Seems easy to add and is easy to use in models after #632 allowing others to try!

```python

from pymc_marketing.prior import Prior
from pymc_marketing.mmm.components.saturation import SaturationTransformation

# pymc_marketing/mmm/transformations
def root_saturation(x, alpha): 
    return x ** alpha

# pymc_marketing/mmm/components/saturation.py
class RootSaturation(SaturationTransformation): 
    def function(self, x, alpha, beta): 
        return beta * root_saturation(x, alpha)

    # More thought required here for sensible priors
    default_priors = {
        "alpha": Prior("Beta", alpha=1, beta=2),
        "beta": Prior("Gamma", mu=1, sigma=1),
    }

# Added below in the dictionary for string access
```

What are your thoughts here? @louismagowan

**Comment by wd60622** [comment]
Not sure if many people use this in practice. Seems like the root function is pretty sensitive to saturating quickly with alpha parameter. 

Might still be an interesting addition to package.

**Comment by louismagowan** [comment]
Yep definitely not a priority transformation in my mind @wd60622  - but your implementation seems v nice anyway!

**Comment by iraur** [comment]
Hey @wd60622, this looks cool - I've opened a PR [here](https://github.com/pymc-labs/pymc-marketing/pull/858) for this and will remove the draft label shortly. 
Thanks!
